soh/soh/Enhancements/randomizer/grassanity.cpp

```diff
@@ -0,0 +1,113 @@
+#include "grassanity.h"
```

Comment:
Could we get these files renamed to ShuffleGrass.cpp/.h? We've recently discussed this and would like to move away from the "sanity" naming. I know there's other options around that are still called that (like fishsanity among others).